dotnet/src/Connectors/Connectors.AI.PaLM/Connectors.AI.PaLM.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Revert the PaLM stuff? I see there's another PR for that...

Reply:
my mistake, that's my another PR.. sorry